target/linux/mpc85xx/p1020/config-default

```diff
@@ -3,6 +3,8 @@ CONFIG_AT803X_PHY=y
 CONFIG_BROADCOM_PHY=y
 CONFIG_CMDLINE_OVERRIDE=y
 CONFIG_CPU_RMAP=y
+CONFIG_CRYPTO_DEV_TALITOS=y
+CONFIG_CRYPTO_DEV_TALITOS2=y
```

Comment:
I am thinking. 

Is it necessary to enable it directly in the kernel or can you use [kmod-crypto-hw-talitos]https://github.com/openwrt/openwrt/blob/416d4483e878fe316ba490c3948850c0aee0b828/package/kernel/linux/modules/crypto.mk#L433)? If it is the first case, this can be separated and done in different pull requests, where you enable it to all mpc85xx subtargets.

Reply:
Thanks, good point. Missed to move that. I'll try it.

Reply:
Fixed, thanks!